aten/src/ATen/native/cuda/Distributions.cu

```diff
@@ -42,15 +42,15 @@
  * we are using curand distributions that utilize curand4 call. curand4 call doesn't have the
  * register spilling problem.
  */
- 
+
```

Comment:
You don't have to fix this, but for future note: try to avoid making unnecessary formatting changes to files that are not related to your PR. That just leads to spurious merge conflicts. If you really hate the formatting, just submit another diff for it (and I promise I'll merge it quick.)